Boost Ctrl/Boost Ctrl/Controller/Matches/MatchesContentViewController.swift

```diff
@@ -108,21 +108,50 @@ class MatchesContentViewController: UIViewController {
 	}
 	
 	@objc func handleRefreshControl() {
-		self.reloadFirebaseData()
-		// Dismiss the refresh control.
-		DispatchQueue.main.async {
-			self.tableView.refreshControl?.endRefreshing()
-			self.tableView.reloadData()
-		}
+        reloadFirebaseData { [weak self] in
```

Comment:
I don't think this will cause a retain cycle but I don't know the code well enough to say for sure so I added weak self just in case.